public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -338,6 +338,9 @@ define(function(require) {
 
       publisher = new Publisher();
       publisher.init(bramble);
+
+      // Initialize the project name UI
+      ProjectRenameUtility.init(appUrl, csrfToken, publisher, bramble);
```

Comment:
I think maybe we should maybe have the order be: `bramble, appUrl, csrfToken, publisher`